test/e2e/reconciler/workspace/controller_test.go

```diff
@@ -196,29 +228,35 @@ func TestWorkspaceController(t *testing.T) {
 				}, metav1.CreateOptions{})
 				require.NoError(t, err, "failed to create workspace shard")
 
-				err = server.expectShard(workspaceShard, func(shard *tenancyv1alpha1.WorkspaceShard) error {
+				t.Logf("Expect the shard to show valid credentials")
+				err = server.rootExpectShard(workspaceShard, func(shard *tenancyv1alpha1.WorkspaceShard) error {
 					if !utilconditions.IsTrue(shard, tenancyv1alpha1.WorkspaceShardCredentialsValid) {
 						return fmt.Errorf("workspace shard %s does not have valid credentials, conditions: %#v", shard.Name, shard.GetConditions())
 					}
 					return nil
 				})
 				require.NoError(t, err, "did not see workspace shard get valid credentials")
 
-				workspace, err := server.client.TenancyV1alpha1().Workspaces().Create(ctx, &tenancyv1alpha1.Workspace{ObjectMeta: metav1.ObjectMeta{Name: "steve"}}, metav1.CreateOptions{})
+				t.Logf("Create a workspace")
+				workspace, err := server.orgKcpClient.TenancyV1alpha1().ClusterWorkspaces().Create(ctx, &tenancyv1alpha1.ClusterWorkspace{ObjectMeta: metav1.ObjectMeta{Name: "steve"}}, metav1.CreateOptions{})
 				require.NoError(t, err, "failed to create workspace")
 
 				server.Artifact(t, func() (runtime.Object, error) {
-					return server.client.TenancyV1alpha1().Workspaces().Get(ctx, workspace.Name, metav1.GetOptions{})
+					return server.orgKcpClient.TenancyV1alpha1().ClusterWorkspaces().Get(ctx, workspace.Name, metav1.GetOptions{})
 				})
 
-				err = server.expect(workspace, func(workspace *tenancyv1alpha1.Workspace) error {
+				t.Logf("Expect workspace to be scheduled to the shard with the base URL stored in the credentials")
+				_, orgName, err := helper.ParseLogicalClusterName(workspace.ClusterName)
```

Comment:
How does `workspace.ClusterName` differ from `workspaceClusterName`?

Reply:
the first is where the ClusterWorkspace is stored, the second the logical cluster for objects inside.

Reply:
Sorry to not have been clear, I was asking for an example of the form of one compared to the other. As far as I know the actual string would be identical.